<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="pantalla">
   <property name="styleSheet">
    <string notr="true">#pantalla{
	background-color:#13173E
}</string>
   </property>
   <widget class="QWidget" name="calculadora" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>751</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> #calculadora{
	background-color:#2596be;
}</string>
    </property>
    <widget class="QPushButton" name="btn1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn3">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn6">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn5">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn8">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>8</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>7</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn9">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>90</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>9</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn0">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>291</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSumar">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>10</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnRestar">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>50</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnMultiplicar">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>90</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>*</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDividir">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>130</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>/</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnIgual">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>10</y>
       <width>71</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string>=</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="txtOperacion">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>10</y>
       <width>271</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="txtResultado">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>120</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>771</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:black</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="btnModo">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white</string>
     </property>
     <property name="text">
      <string>Modo claro</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="table">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>321</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#F2F2DD</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>40</y>
      <width>261</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#F2F2DD</string>
    </property>
    <widget class="QLineEdit" name="txtMarca">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtTalla">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtPrecio">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>70</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>MARCA</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>TALLA</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>PRECIO</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAnadir">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>241</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>AÃ‘ADIR</string>
     </property>
    </widget>
   </widget>
   <zorder>calculadora</zorder>
   <zorder>frame</zorder>
   <zorder>widget</zorder>
   <zorder>table</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
